pkg/operator/replacement_starter.go

```diff
@@ -140,6 +143,7 @@ func CreateOperatorInputFromMOM(ctx context.Context, momInput libraryapplyconfig
 		apiextensionClient:           apiextensionClient,
 		eventRecorder:                eventRecorder,
 		clock:                        momInput.Clock,
+		featureGateAccessor:          momFeatureGateAccessor(),
```

Comment:
It might make the creation of `authenticationOperatorInput` clearer if the enabled/disabled featuregates are obvious on the constructor; something like:
```suggestion
		featureGateAccessor:          staticFeatureGateAccessor([]ocpconfigv1.FeatureGateName{features.FeatureGateExternalOIDC}, []ocpconfigv1.FeatureGateName{}),


```

Just an idea though, I'll leave it up to you to decide what you prefer.